content/post/building-a-data-practice.md

```diff
@@ -0,0 +1,85 @@
+
+While mapping the data infrastructure, start thinking about what is missing, what you would like to change, and what the future infrastructure might look like. These ideas can go on your roadmap, which I cover next.
+
+### Plan your roadmap
+One of the perennial Data team struggles is balancing the short term and the long term. You will be busy right away, pulled in multiple directions, but you should take the time to form opinions about the future. How would you like to evolve data tooling, infrastructure, processes, and people?
+
+Think about what is missing, what can be improved, and how they align with the organization's priorities. Are you using Google Analytics for tracking? That might be fine for now, but many businesses will eventually need something more powerful. How are reports being run? If data is primarily accessed and explored in spreadsheets, you should think about options for BI tools. Who should be hired next? A data analyst? A data engineer?
+
+Form opinions about your roadmap now so you will be prepared to advocate for it whenever it comes up. What will you say when someone in marketing decides they want to integrate a new service, or when an engineer starts to build a data tool in house, or when the founders raise more funding and ask you how to spend it?
+
+### Create a data dictionary
+As you learn things, especially things that are surprising, write them down. Think about what the next data hire will need to know. Avoid the urge to make this fancy — a Google doc is fine.
+
+The concept of a data dictionary is a bit controversial. What do you include? What don't you include? How do you maintain it? For more mature organizations, I recommend considering [Carl Anderson's suggestions](https://medium.com/@leapingllamas/data-dictionary-a-how-to-and-best-practices-a09a685dcd61) for using a data dictionary to create a shared understanding around key metrics and vocabulary.
+
+In this case, though, a data dictionary is primarily for you and future data hires. Even though the business and its data will change over time, there are still key pieces of information that are useful to document. Are timezones in UTC or your organization's time? What are some core pieces of business logic? How are key metrics calculated? Are there important database tables or column names that are not self-explanatory?
+
+At Kickstarter, we called this document *Common Data Gotchas*, which is a good way to think about it.
+
+### Use a project management tool
+Start tracking all the work you do, even quick data requests. Write down the question, the results, and how you got them. Encourage others in your organization to interact with you in this tool. Documenting your work in this way will be important for reproducibility, it will be good process as your data team grows, and it will help with visibility and transparency, demonstrating your impact on the business.
```

Comment:
I think you could flesh the "why" out a bit more here.

Reply:
Yes - and maybe something like "be careful not to make this process burdensome for stakeholders too early on"